README.md

```diff
@@ -1,4 +1,7 @@
-# ert
+<div style="text-align: center;">
+<img width="400" src="src/ert/gui/resources/gui/img/ert_icon.svg">
```

Comment:
This is way too big

Reply:
Using the same scale as numpy:

![image](https://github.com/equinor/ert/assets/44577479/14b95f50-11e2-4159-8a20-b046385c241c)